ci/requirements-3.5_OSX.run

```diff
@@ -1,5 +1,5 @@
 pytz
-numpy
+numpy=1.10.4
```

Comment:
Pls only include changes you've made(see other PRs). You need rebase:

git checkout master
git pull upstream
git checkout <your branch>
git rebase -i master